### PR TITLE
Updating Rules

### PR DESCRIPTION
**Pull request by saramaebee**

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -39,73 +39,53 @@
...
             "name": "Asking For Help",
-            "triggers": ["1", "ask", "help"],
-            "description": "Actually ask your question, don't just ask for \"help\"."
-        },
-        {
-            "name": "Explain What's Wrong",
-            "triggers": ["2", "work", "explain"],
-            "description": "Don't ask why your code doesn't \"work\"."
+            "triggers": ["0", "ask", "help", "details"],
+            "description": "Help us to help you, instead of just saying \"my code doesn't work\" or \"can someone help me.\""
```
Comment (LamboCreeper) [inline src/config.json:43]: The full stop (or period, Americans smh...) should be between the two quotes.

File: src/config.json [hunk 2]
```diff
@@ -39,73 +39,53 @@
...
+            "name": "Be Nice",
+            "triggers": ["3", "respect", "nice"],
+            "description": "Be respectful; no personal attacks, sexism, homophobia, transphobia, racism, hate speech or other disruptive behaviour."
         },
         {
             "name": "No Advertising",
-            "triggers": ["7", "advertising"],
-            "description": "Don't advertise, it's as simple as that."
+            "triggers": ["4", "advertising", "advertise", "advertisement", "ad"],
+            "description": "Don't advertise. If you're not sure whether it would be considered advertising or not, ask a moderator."
```
Comment (LamboCreeper) [inline src/config.json:63]: Not sure the comma is needed?

File: src/config.json [hunk 3]
```diff
@@ -39,73 +39,53 @@
...
         },
         {
             "name": "Use The Right Channel",
-            "triggers": ["8", "channel"],
-            "description": "Stick to the appropriate channels. Feel free to ask in [#general](https://discord.gg/qZfADKn) if you're not sure where to ask something."
+            "triggers": ["5", "channel"],
+            "description": "Stick to the correct channels. If you're unsure which channel to put your question in, you can ask in [#general](https://codesupport.dev/discord) which channel is best for your question."
+		},
+		{
```
Comment (LamboCreeper) [inline src/config.json]: Funky formatting.

File: src/config.json [hunk 4]
```diff
@@ -39,73 +39,53 @@
...
+            "name": "Illegal/Immoral Tasks",
+            "triggers": ["6", "illegal", "immoral"],
+            "description": "Don't ask for help with illegal or immoral tasks. Doing so not only risks your continued participation in this community but is in violation of Discord's TOS and can get your account banned."
+		},
+		{
```
Comment (LamboCreeper) [inline src/config.json]: Funky formatting.

File: src/config.json (hunk 2 above)
Comment (saramaebee) [reply src/config.json:63]: to be gramatically correct, it is

File: src/config.json (hunk 1 above)
Comment (saramaebee) [reply src/config.json:43]: it is

Comment (saramaebee) [reply src/config.json:43]: you're looking at the red

**Review by LamboCreeper — APPROVED** [review]

**Pull request merged**